src/crc32c_arm64.cc

```diff
@@ -24,17 +25,31 @@
 #define KBYTES 1032
 #define SEGMENTBYTES 256
 
+static uint64_t LoadU64(const uint8_t in[sizeof(uint64_t)]) {
```

Comment:
Can you use and extend crc32c_read_le.h?

I think that ReadUint32LE() and ReadUint64LE() there work as you need them to, and we checked that they're compiled as efficiently as memcpy on little-endian architectures.

You can add a ReadUint16LE() with tests, and then use it here.

Reply:
Ah nice! Done. See commit message of https://github.com/google/crc32c/pull/65/commits/1b4eb1a281c99f2bb772ad0055b3e50e75770673 for some notes on what I changed there.